tests/specs/lint/lint_plugin_source_code/main.ts

```diff
@@ -0,0 +1,2 @@
+const value = "unfixed";
+console.log(value);
```

Comment:
Can you update this file to contain more text including things like ligatures and emojis? Just to better stress text the utf16 codepoint handling.